packages/utilities/gel-foundations/src/scripts/sinhalese-noto-serif.js

```diff
@@ -0,0 +1,214 @@
+const sinhaleseNotoSerifTypography = {
+  atlas: {
+    groupA: {
+      fontSize: '78',
+      lineHeight: '84',
+    },
+    groupB: {
+      fontSize: '96',
+      lineHeight: '104',
+    },
+    groupD: {
+      fontSize: '140',
+      lineHeight: '148',
+    },
+  },
+  elephant: {
+    groupA: {
+      fontSize: '60',
+      lineHeight: '64',
+    },
+    groupB: {
+      fontSize: '78',
+      lineHeight: '84',
+    },
+    groupD: {
+      fontSize: '116',
+      lineHeight: '124',
+    },
+  },
+  imperial: {
+    groupA: {
+      fontSize: '50',
+      lineHeight: '54',
+    },
+    groupB: {
+      fontSize: '64',
+      lineHeight: '72',
+    },
+    groupD: {
+      fontSize: '96',
+      lineHeight: '104',
+    },
+  },
+  royal: {
+    groupA: {
+      fontSize: '40',
+      lineHeight: '44',
+    },
+    groupB: {
+      fontSize: '52',
+      lineHeight: '60',
+    },
+    groupD: {
+      fontSize: '76',
+      lineHeight: '84',
+    },
+  },
+  foolscap: {
+    groupA: {
+      fontSize: '32',
+      lineHeight: '36',
+    },
+    groupB: {
+      fontSize: '40',
+      lineHeight: '44',
+    },
+    groupD: {
+      fontSize: '56',
+      lineHeight: '60',
+    },
+  },
+  canon: {
+    groupA: {
+      fontSize: '25',
+      lineHeight: '32',
+    },
+    groupB: {
+      fontSize: '31',
+      lineHeight: '40',
+    },
+    groupD: {
+      fontSize: '42',
+      lineHeight: '58',
+    },
+  },
+  trafalgar: {
+    groupA: {
+      fontSize: '20',
+      lineHeight: '24',
+    },
+    groupB: {
+      fontSize: '27',
+      lineHeight: '36',
+    },
+    groupD: {
+      fontSize: '32',
+      lineHeight: '44',
+    },
+  },
+  paragon: {
+    groupA: {
+      fontSize: '19',
+      lineHeight: '24',
+    },
+    groupB: {
+      fontSize: '23',
+      lineHeight: '32',
+    },
+    groupD: {
+      fontSize: '28',
+      lineHeight: '38',
+    },
+  },
+  doublePica: {
+    groupA: {
+      fontSize: '18',
+      lineHeight: '22',
+    },
+    groupB: {
+      fontSize: '21',
+      lineHeight: '30',
+    },
+    groupD: {
+      fontSize: '24',
+      lineHeight: '34',
+    },
+  },
+  greatPrimer: {
+    groupA: {
+      fontSize: '18',
+      lineHeight: '22',
+    },
+    groupB: {
+      fontSize: '19',
+      lineHeight: '24',
+    },
+    groupD: {
+      fontSize: '19',
+      lineHeight: '24',
+    },
+  },
+  bodyCopy: {
+    groupA: {
+      fontSize: '18',
+      lineHeight: '28',
+    },
+    groupB: {
+      fontSize: '18',
+      lineHeight: '28',
+    },
+    groupD: {
+      fontSize: '18',
+      lineHeight: '28',
+    },
+  },
+  pica: {
+    groupA: {
+      fontSize: '17',
+      lineHeight: '28',
+    },
+    groupB: {
+      fontSize: '17',
+      lineHeight: '24',
+    },
+    groupD: {
+      fontSize: '22',
```

Comment:
```suggestion
      fontSize: '17',
```

Reply:
Incorrect compared to the table on the issue:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/9645462/133291671-82a1e7be-313d-4cdf-ace9-89f8df9bfb3f.png">